scipy/spatial/ckdtree.pyx

```diff
@@ -1518,6 +1520,58 @@ cdef class cKDTree:
         else:
             raise ValueError('Invalid output type')
 
+    # -----
+    # remove
+    # -----
+
+    def remove(self, np.intp_t index):
```

Comment:
Can you add a wrapper to [`KDTree`](https://github.com/scipy/scipy/blob/8e30f7797bd1ee442f4f1a25172e4402521c1e16/scipy/spatial/kdtree.py#L182) which just calls `super().remove(index)` but uses the name `KDTree` in the docstring.